examples/wavenet/train.py

```diff
@@ -94,15 +104,16 @@
 
 # Updater and Trainer
 updater = chainer.training.StandardUpdater(
-    train_iter, optimizer, device=args.gpu)
+    train_iter, optimizer, device=device)
 trainer = chainer.training.Trainer(
     updater, (args.epoch, 'epoch'), out=args.out)
 
 # Extensions
 snapshot_interval = (args.snapshot_interval, 'iteration')
 display_interval = (args.display_interval, 'iteration')
-trainer.extend(extensions.Evaluator(valid_iter, model, device=args.gpu))
-trainer.extend(extensions.dump_graph('main/loss'))
+trainer.extend(extensions.Evaluator(valid_iter, model, device=device))
+if device.xp is not chainerx:
```

Comment:
A minor but should we add a TODO similar to https://github.com/chainer/chainer/pull/6739/files#diff-30d5da2c4d62b2408b6ff6087fdd3ef3R104 ?

Reply:
Ditto. Please review once again.